exercises/connect/connect.test.ts

```diff
@@ -0,0 +1,108 @@
+import Board from './connect'
```

Comment:
An you name this Connect or ConnectBoard ?

Reply:
`ConnectBoard` looks good for me.